sdk/attestation/azure-security-attestation/azure/security/attestation/aio/_administration_client_async.py

```diff
@@ -4,320 +4,692 @@
     :type credential: :class:`~azure.core.credentials_async.AsyncTokenCredential`
-    :keyword AsyncPipelineClient pipeline: If omitted, the standard pipeline is used.
-    :keyword AsyncHttpTransport transport: If omitted, the standard pipeline is used.
-    :keyword list[AsyncHTTPPolicy] policies: If omitted, the standard pipeline is used.
+    :param str endpoint: The attestation instance base URI, for example https://mytenant.attest.azure.net.
+    :keyword str signing_key: PEM encoded signing key to be used for all
+        operations.
+    :keyword str signing_certificate: PEM encoded X.509 certificate to be used for all
+        operations.
+    :keyword bool validate_token: If True, validate the token, otherwise return the token unvalidated.
+    :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+        if the token is invalid, the `validation_callback` function should throw
+        an exception.
+    :paramtype validation_callback: ~typing.Callable[[~azure.security.attestation.AttestationToken, ~azure.security.attestation.AttestationSigner], None]
+    :keyword bool validate_signature: If True, validate the signature of the token being validated.
+    :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+    :keyword str issuer: Expected issuer, used if `validate_issuer` is true.
+    :keyword float validation_slack: Slack time for validation - tolerance applied
+        to help account for clock drift between the issuer and the current machine.
+    :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+    :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
+
+    If the `signing_key` and `signing_certificate` parameters
+    are provided, they will be applied to the following APIs:
+
+    * :py:func:`set_policy`
+    * :py:func:`reset_policy`
+    * :py:func:`add_policy_management_certificate`
+    * :py:func:`remove_policy_management_certificate`
+
+    .. note::
+        The `signing_key` and `signing_certificate` parameters are a pair. If one
+        is present, the other must also be provided. In addition, the public key
+        in the `signing_key` and the public key in the `signing_certificate` must
+        match to ensure that the `signing_certificate` can be used to validate an
+        object signed by `signing_key`.
+
+    .. tip::
+        The `validate_token`, `validation_callback`, `validate_signature`,
+        `validate_expiration`, `validate_not_before_time`, `validate_issuer`, and
+        `issuer` keyword arguments are default values applied to each API call within
+        the :py:class:`AttestationAdministrationClient` class. These values can be
+        overridden on individual API calls as needed.
 
     For additional client creation configuration options, please see https://aka.ms/azsdk/python/options.
```

Comment:
Like in the sync AttestationClient, I'd use a named link here. Also, this line doesn't seem to be at the end of the sync AttestationAdministrationClient docstring -- is that intentional?
```suggestion
    For additional client creation configuration options, please see `Python Request
    Options <https://aka.ms/azsdk/python/options>`_.
```